src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasCountrySlicer.java

```diff
@@ -141,18 +243,50 @@ public Atlas slice()
         }
     }
 
+    /**
+     * @param relation
+     *            The {@link Relation} for which to create seeds for
+     * @return an array of identifiers corresponding to the first unused identifier for each
+     *         relation member in the given Relation
+     */
+    private long[] createIdentifierSeeds(final Relation relation)
+    {
+        // Assume any member can be split, so create a seed for each one
+        final long[] seeds = new long[relation.members().size()];
```

Comment:
Dumb question: is it possible to have a `relation` with no members?

Reply:
No such thing as a dumb question! It's not possible - one of the Atlas relation constraints is that we cannot have a Relation without any members. See [here](https://github.com/osmlab/atlas/blob/1dbcbe5818d018f7c919b51d8ea82902ec1a5fa7/src/main/java/org/openstreetmap/atlas/geography/atlas/packed/PackedAtlas.java#L923) for reference code. So at this point, we can safely assume there will be at least one member.